tests/astrapy/test_db_dml.py

```diff
@@ -324,6 +324,193 @@ def test_insert_many(writable_vector_collection: AstraDBCollection) -> None:
     assert isinstance(list(inserted_ids - {_id0, _id2})[0], str)
 
 
+@pytest.mark.describe("batched_concurrent_insert_many")
+def test_batched_concurrent_insert_many(
+    writable_vector_collection: AstraDBCollection,
+) -> None:
+    _id0 = str(uuid.uuid4())
+    _id2 = str(uuid.uuid4())
+    documents: List[API_DOC] = [
```

Comment:
This long list could be replaced by a repeated element.
Something like
```python
[{"_id": str(uuid.uuid4()), "name": "Abba"} for i in range(30)]
```

Reply:
I have rewritten the whole test code.